packages/react-web/CHANGELOG.md

```diff
@@ -1,5 +1,296 @@
 # @lens-protocol/react-web
 
+## 2.0.0
+
+### Minor Changes
+
+- a98f6ad4e: **feat:** allow to overwrite all onchain transactions to be self-funded on the config level
+- cfc03dff8: **feat:** implements `useOpenAction` hook
+- 4630efb96: add report publication
+- 25b7ef512: add hide publicaiton
+- 493895b8c: **feat:** self-funded `useFollow`
+- 9aa0fb780: **chore:** unified implementation and naming of `isValidHandle` helper among react and client SDKs. deprecated `isValidProfileHandle` in the client sdk.
+  **feat:** added `useValidateHandle` hook
+- dd2f7d246: feat: adds self-funded support for unfollow
+- d255b3627: **feat:** Added `useDismissRecommendedProfiles` hook
+- a929c0f6e: **feat:** implements `useCreatePost` hook
+- 5ecead02d: **breaking:** Remove all what was marked as deprecated. See the detailed list below. Prepare for the major release.
```

Comment:
Can we verify this block renders well in MD?

Reply:
you can see how it renders already here:

https://github.com/lens-protocol/lens-sdk/blob/chore/stable-release-2.0.0/packages/react-web/CHANGELOG.md

Reply:
Sweet!